Libraries/Lists/FlatList.js

```diff
@@ -625,16 +634,20 @@ class FlatList<ItemT> extends React.PureComponent<Props<ItemT>, void> {
   };
 
   render(): React.Node {
-    const {numColumns, columnWrapperStyle, ...restProps} = this.props;
+    const {columnWrapperStyle, ...restProps} = this.props;
 
     return (
       <VirtualizedList
         {...restProps}
+        numColumns={undefined}
```

Comment:
What's the reasoning for this? Are you preventing `numColumns` from being passed in? If so, I'd keep this in the object extraction on line 637 and do something like: 

```
const {numColumns, columnWrapperStyle, removeClippedSubviews: _removeClippedSubviews,  ...restProps} = this.props;

<VirtualizedList
....
removeClippedSubviews={removeClippedSubviewsOrDefault(_removeClippedSubviews)}
... />
```

Reply:
Yes I wanted to not pass `numColumns` into the VirtualizedList, because it'll break the existing tests if i do. So my conclusion was that it should not be passed. 

Unfortunately, I don't quite understand your suggestion yet. Should I use the accessor function for the `numColumns` variable? That would lead to the variable being passed on again and the tests failing, wouldn't it? Can you help me further where my thinking error lies?

Reply:
Aaah yes you are right! 💯 Thanks for pointing this and giving the suggestion with the alias in the destructuring. I really like that! 
@lunaleaps I already learned a lot from you in this review, thanks!